### PR TITLE
Update frontend dependencies

### PR DESCRIPTION
**Pull request by SenseException**
Follow up to #432. 

This PR introduces updates to the frontend dependencies while keeping compatibility to the website code without too much chnages.

### CODE REVIEW

**Comment by SenseException** [comment]
I've deployed the changes of this branch to staging. https://staging.doctrine-project.org/

File: source/js/main.js [hunk 1]
```diff
@@ -57,8 +57,8 @@ export default function() {
       $('li:first', $(this)).addClass('selected');
     });
 
-    $('[data-toggle="$: \'jquery\',\n' +
-        '  jQuery: \'jquery\'"]').tooltip();
+    // $('[data-toggle="$: \'jquery\',\n' +
+    //     '  jQuery: \'jquery\'"]').tooltip();
```
Comment (greg0ire) [inline source/js/main.js:61]: Why is this commented out?

Comment (SenseException) [reply source/js/main.js:61]: The tooltip Plugin wasn't available by default anymore and I honestly don't know where it's used.

Comment (greg0ire) [reply source/js/main.js:61]: Let's remove that piece of code and use your comment as a commit message then?

Comment (SenseException) [reply source/js/main.js:61]: Sounds link you didn't find anything on staging too. Okay, I'll create a separate commit for the removal.

Comment (greg0ire) [reply source/js/main.js]: Yup. Still seems to work great :)

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @SenseException !

**Pull request merged**